tools/ipfs-deploy.js

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env node
+
+/**
+ * Runs the fleek deploy process based on nx:affected, but only for sites
+ * that have a .fleek.json file
+ */
+
+// Fleek CLI requires this variable to be set
+if (!process.env['FLEEK_API_KEY']) {
+  console.error('Error: FLEEK_API_KEY must be set');
+  process.exit(1);
+}
+
+const { existsSync } = require('fs');
```

Comment:
nitpick (non-blocking): I like requires at the top of a file so they can be easily seen as the dependecies.

Reply:
Good catch, not sure how this ended up here.